be/src/olap/base_compaction.cpp

```diff
@@ -174,7 +172,10 @@ OLAPStatus BaseCompaction::run() {
     _release_base_compaction_lock();
 
     LOG(INFO) << "succeed to do base compaction. tablet=" << _tablet->full_name()
-              << ", base_version=" << _new_base_version.first << "-" << _new_base_version.second;
+              << ", base_version=" << _new_base_version.first << "-" << _new_base_version.second
+              << ". elapsed time of doing base compaction"
+              << ", time=" << stage_watch.get_elapse_time_us() / (100000.0) << "s";
```

Comment:
I think there is need to  add a get_elapse_time_s() function?

Reply:
OK